app/views/comments/_comments.html.erb

```diff
@@ -34,11 +34,13 @@
       <td><%= distance_of_time_in_words(comment.created_at, Time.current, { include_seconds: false, scope: 'datetime.time_ago_in_words' }) %></td>
       <td>
         <% if comment.status == 4 && current_user &. can_moderate? %>
+              Status: <%= comment.status_value %>
               <a class="btn btn-outline-secondary btn-sm" href="/admin/publish_comment/<%= comment.id %>"><%= translation('dashboard.moderate.approve') %></a>
               <a class="btn btn-outline-secondary btn-sm" href="/admin/mark_comment_spam/<%= comment.id %>"><%= translation('dashboard.moderate.spam') %></a>
         <% elsif current_user &. can_moderate? %>
+            Status: <%= comment.status_value  %>
             <a rel="tooltip" title="Mark as spam" class="btn btn-sm btn-outline-secondary btn-flag-spam-<%= comment.id %>" data-confirm="Are you sure? The user will no longer be able to log in or publish, and their content will be hidden except comments." href="/admin/mark_comment_spam/<%= comment.id %>">
-              <i class="fa fa-ban"></i>
+            <i class="fa fa-ban"></i>
```

Comment:
Please add the indented space as before.Thanks

Reply:
@sssash18 Thanks

Reply:
The tests failing doesn't seem relevant to your changes. I think you must close and reopen the PR to restart travis.